packages/nadle/test/__snapshots__/errors.test.ts.snap

```diff
@@ -25,3 +27,27 @@ Hello from nadle!
 [log] 
 For more details, re-run the command with the <Yellow>--stacktrace</Yellow> option to display the full error and help identify the root cause.
 `;
+
+exports[`when a task fails > should report correctly > stdout 2`] = `
```

Comment:
I wonder why there is no change in the test file but new snapshot is introduced.